lib/ErrorManager.php

```diff
@@ -19,14 +20,23 @@ public function __construct(Configuration $configuration)
         $this->configuration = $configuration;
     }
 
-    public function error(string $message): void
+    public function error(string $message, ?Throwable $throwable = null): void
     {
         $this->errors[] = $message;
 
         if ($this->configuration->isAbortOnError()) {
+            // throw the actual throwable if one is available
+            if ($throwable !== null) {
+                throw $throwable;
+            }
```

Comment:
When the error comes from an actual exception, this is helpful because we can see the original stacktrace. I have only updated one place in the code so far to pass the exception.